src/managers/channel-manager.js

```diff
@@ -3,18 +3,35 @@
 var Channel = require('../service/channel-service');
 
 /**
- * Generic wrapper around $.cometd. This provides a few nice features the default cometd wrapper doesn't
+ * ## Channel Manager
  *
- * - Automatic re-subscription to channels if you lose your connection in-between
- * - Online/ Offline notifications
- * - 'Events' for cometd notifications, instead of having to listen on specific meta channels
+ * The Epicenter platform provides a push channel, which allows you to publish and subscribe to messages within a [project](../../../glossary/#projects), [group](../../../glossary/#groups), or [multiplayer world](../../../glossary/#world). There are two main use cases for the channel: event notifications and chat messages.
```

Comment:
just for context ... i'm a bit unclear how/why you would be here if you weren't using epicenter. 

i removed the first reference but kept the paragraph "while you can work directly with the channel mgr most often you'll use epicenter channel manager...".